decidim-admin/config/locales/en.yml

```diff
@@ -463,6 +465,17 @@ en:
           homepage_highlighted_content_banner_title: Highligted content banner
           layout_appearance_title: Edit layout appearance
           omnipresent_banner_appearance_title: Edit omnipresent
+      participatory_space_private_users:
+        create:
+          error: There was an error adding a private user for this participatory space.
+          success: Private Participatory Space user access created successfully.
+        destroy:
+          success: Private Participatory Space user access destroyed successfully.
+        index:
+          title: Private users Participatory Space
```

Comment:
- `Participatory space private user`
- `Private Participatory Space user`
- `Private users Participatory Space`

Can we stick to one single use? These are taken from additions in this file 😕

Reply:
`Participatory space private user`